lib/DDG/Spice/Aur.pm

```diff
@@ -1,30 +1,39 @@
 package DDG::Spice::Aur;
-
+# ABSTRACT: Write an abstract here
+# Start at https://duck.co/duckduckhack/spice_overview if you are new
+# to instant answer development
 use strict;
 use DDG::Spice;
 
-primary_example_queries "aur powermate";
-secondary_example_queries "archlinux package 9base-git";
-description "Lookup packages from the Archlinux user repository";
-name "AUR";
+# Caching - https://duck.co/duckduckhack/spice_advanced_backend#caching
+spice is_cached => 1; 
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "Aur";
+source "aur.archlinux.org";
+icon_url "http://upload.wikimedia.org/wikipedia/commons/thumb/a/a5/Archlinux-icon-crystal-64.svg/1000px-Archlinux-icon-crystal-64.svg.png";
+description "Provides hook into Arch User Repository API";
+primary_example_queries "aur mopidy", "aur python2-twisted";
```

Comment:
Look like there's no API results for "aur python2-twisted" -- let's change that to a query that works :) 

We usually like to have passing tests for the example queries too.